Doc/library/zipfile.rst

```diff
@@ -637,22 +641,41 @@ file:
    .. versionadded:: 3.8
       The *strict_timestamps* keyword-only argument
 
+   .. versionadded:: 3.9
+      The *follow_symlinks* keyword-only argument
+
 
 Instances have the following methods and attributes:
 
 .. method:: ZipInfo.is_dir()
 
-   Return ``True`` if this archive member is a directory.
+   Return :const:`True` if this archive member is a directory.
 
+   .. versionadded:: 3.6
    This uses the entry's name: directories should always end with ``/``.
 
-   .. versionadded:: 3.6
+   .. versionchanged:: 3.9
+   Use the filemode contained in external file attributes.
+
+.. method:: ZipInfo.is_reg()
 
+   Return :const:`True` if this archive member is a regular file.
+
+.. method:: ZipInfo.is_file()
```

Comment:
I don't see the point in creating both `is_reg` and `is_file`. Perhaps you could just keep `isfile()` and not add `isreg()`? (unless there's another reason to keep them both)

Reply:
This was ot mirror `TarInfo.isreg()` and `TarInfo.isfile()`, but maybe it's better with only `isfile()` indeed

Reply:
Yeah, I think only `is_file()` is needed.

Reply:
`is_reg()` removed in ca5e302